docker/sage/services/otel-collector.yml

```diff
@@ -0,0 +1,18 @@
+services:
+  sage-otel-collector:
+    image: ghcr.io/sage-bionetworks/sage-otel-collector:${SAGE_VERSION:-local}
+    container_name: sage-otel-collector
+    restart: always
+    networks:
+      - sage
+    ports:
+      - '1888:1888' # pprof extension
+      - '8888:8888' # Prometheus metrics exposed by the collector
+      - '8889:8889' # Prometheus exporter metrics
+      - '13133:13133' # health_check extension
+      - '4317:4317' # OTLP gRPC receiver
```

Comment:
For my purpose only 13133 and 4318 are needed. I am sending data over http instead of grpc

If you are planning to use this for OC, then some more ports are needed

Reply:
This is addressed in #2909.